packages/conf-libclang/conf-libclang.19/opam

```diff
@@ -0,0 +1,59 @@
+opam-version: "2.0"
+maintainer: "Thierry Martinez <martinez@nsup.org>"
+authors: "The LLVM team"
+homepage: "http://llvm.org"
+bug-reports: "https://llvm.org/bugs/"
+license: "MIT"
+build-env: [
+  [HOMEBREW_NO_AUTO_UPDATE = "1"]
+]
+build: [
+  ["bash" "-ex" "configure.sh" version] # pass pkg var '19' to test <= 19.0.x
+]
+depends: [
+  "conf-bash" {build}
+]
+depexts: [
+  ["llvm@19"] {os-distribution = "homebrew" & os = "macos"}
+  ["llvm-19"] {os-distribution = "macports" & os = "macos"}
+  ["llvm" "clang"] {os-distribution = "arch"} #currently llvm is 21.1.6-1 https://archlinux.org/packages/extra/x86_64/llvm/
+  # Ubuntu
+    ["libclang-19-dev" "libclang-cpp19-dev" "llvm-19-dev"]         {os-distribution = "ubuntu" & os-version > "24.04" }
+  ["libclang-dev" "libclang-cpp-dev" "llvm-dev"]         {os-distribution = "ubuntu" & os-version > "22.10" & os-version < "24.04"}
+  ["libclang-15-dev" "libclang-cpp15-dev" "llvm-15-dev"] {os-distribution = "ubuntu" & os-version = "22.10"}
+  ["libclang-14-dev" "libclang-cpp14-dev" "llvm-14-dev"] {os-distribution = "ubuntu" & os-version = "22.04"}
+  # Debian
+  ["libclang-19-dev" "libclang-cpp19-dev" "llvm-19-dev"]         {os-distribution = "debian" & os-version >= "12"}
+  ["libclang-dev" "libclang-cpp-dev" "llvm-dev"]         {os-distribution = "debian" & os-version = "11"}
+  # Derivatives or older versions
+  ["libclang-19-dev" "llvm-19-dev"] {
+     (os-family = "debian" | os-family = "ubuntu") &
+     !(os-distribution = "debian" & os-version >= "11") &
+      !(os-distribution = "ubuntu" & os-version >= "22.04")}
+  # Alpine
+  ["clang19-dev" "llvm19-dev" "clang19-static"] {os-distribution = "alpine" & os-version >= "3.21"}
+  ["clang-dev" "llvm-dev" "clang-static"] {os-distribution = "alpine" & os-version < "3.21"}
+  ["clang-devel" "llvm-devel" "zlib-devel" "llvm-static"] {os-distribution = "centos"}
+  ["clang-devel" "llvm-devel" "zlib-devel" "redhat-rpm-config"] {os-family = "fedora"}
+  ["clang-devel" "llvm-devel" "zlib-devel" "llvm-static"] {os-distribution = "ol" & os-version >= "8"}
+  ["clang-devel" "llvm-devel" "zlib-devel"] {os-family = "suse"}
+  ["clang-devel" "llvm-devel" "zlib-ng-compat-devel"] {os-family = "opensuse"}
```

Comment:
I also see OpenSuse Tumbleweed failing the test because the installed 21 is greater than "19 or less",
without anything lower available. Probably also a case for `x-ci-accept-failures`.